src/views/Books/BookDetail.jsx

```diff
@@ -1,18 +1,28 @@
+import { Link, useParams } from 'react-router-dom'
 import { useEffect, useState } from 'react'
 import Book from '../../components/book/Book'
 import { getBookById } from '../../services/books'
 
 function BookDetail() {
-  const id = 1 // TODO: Use id from route
+  // const id = useParams() // TODO: Use id from route
   const [book, setBook] = useState(null)
+  const { booksId } = useParams()
 
   useEffect(() => {
-    getBookById(id).then(({ data }) => setBook(data))
-  }, [id])
+    getBookById(booksId).then(({ data }) => setBook(data))
+  }, [booksId])
 
   if (!book) return <h3>Loading book...</h3>
 
-  return <Book book={book} showDetail />
+  return (
+    <>
+      <Book book={book} showDetail />
+      <p>Book Detail: {booksId} </p>
+      <p>
+        <Link to="../books">Back to Catalog</Link>
```

Comment:
because Links are essentially an `<a>` tag and not a file path this should be `<Link to='/books'>`